src/Corax/IndexWriter.cs

```diff
@@ -125,27 +146,82 @@ public long GetNumberOfEntries()
             return Transaction.LowLevelTransaction.RootObjects.ReadInt64(IndexWriter.NumberOfEntriesSlice) ?? 0;
         }
 
-        private void InsertToken(ByteStringContext context, ref IndexEntryReader entryReader, int tokenField, Dictionary<Slice, List<long>> field, long entryId)
+        [SkipLocalsInit]
+        private unsafe void InsertAnalyzedToken(ByteStringContext context, ref IndexEntryReader entryReader, int tokenField, Dictionary<Slice, List<long>> field, long entryId)
         {
+            _analyzer.GetOutputBuffersSize(512, out int bufferSize, out int tokenSize);
```

Comment:
Where did the `512` come from?

Reply:
This number is the number you gave me when we talked... I was worried about multistep text tokenization and you told me that the longest term was going to be no longer than 512 bytes.

Reply:
Yeah, needs to be extracted to a `const`.

Reply:
@redknightlois can you please extract it to the const with some meaningful name

Reply:
Done